src/js/utils/ServiceUtil.js

```diff
@@ -8,26 +8,37 @@ const getFindPropertiesRecursive = function (service, item) {
 
       return memo;
     }
-    memo[subItem] = service.get(subItem) || item[subItem].default;
+    memo[subItem] = item[subItem].getter(service) || item[subItem].default;
 
     return memo;
   }, {});
 };
 
 const ServiceUtil = {
   createServiceFromFormModel: function (formModel) {
-    let values = Object.keys(formModel).reduce(function (memo, section) {
-      Object.keys(formModel[section]).forEach(function (attribute) {
-        memo[attribute] = formModel[section][attribute];
-      });
+    let definition = {};
 
-      return memo;
-    }, {});
+    if (formModel.General != null) {
```

Comment:
maybe out of scope for this PR but I'd like us to enforce some checks on the received `formModel` so that we don't explode if it's not an object. 

We should really start considering `Flow` static typing...